README.md

```diff
@@ -32,34 +49,6 @@ Paths:
  mod/*/features/step_definitions  # cucumber step definitions
 ```
 
-Updating Test DB
-----
-Run `rake wikirate:test:reseed_data` to prepare test database.
```

Comment:
this looked mostly wrong to me (I don't think the initial command even exists).  Is there anything here we need to preserve?

Reply:
that looks outdated to me